### PR TITLE
Segments and Targeting block

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZEE-3282, https://jira.ez.no/browse/EZEE-3283
| Versions      | master (3.2)

### CODE REVIEW

File: docs/site_organization/working_with_page.md [hunk 1]
```diff
@@ -117,6 +118,18 @@
         Settings and function of custom-made blocks of content depend on their design.
         For details on adjusting and using them, contact your website administrator.
 
+    ### Targeting block
+    
+    In the **Basic** tab, perform the following actions:
```
Comment (dabrt) [inline docs/site_organization/working_with_page.md:123]: Do you have to perform the actions below in a specific order? Is so, I would make this a numbered list. A bulleted list implies that actions can be performed in random order.

File: docs/site_organization/working_with_page.md [hunk 2]
```diff
@@ -117,6 +118,18 @@
...
+    
+    - In the **Name** box, provide the name of the block.
+    - In the **Select default content** section, click **Select content**, navigate through the content
+    and select the default Content item that will be displayed when no priority rules are valid.
+    - In the **Setup segment and content matching priority rules** section, select a Segment Group and a Segment,
+    then click **Select content** and navigate to the Content item you want to display for the selected group.
```
Comment (dabrt) [inline docs/site_organization/working_with_page.md:129]:
```suggestion
    then click **Select content** and navigate to the Content item that you want to display for the selected group.
```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: docs/site_organization/organizing_the_content.md [hunk 3]
```diff
@@ -210,3 +210,17 @@ You can bookmark any Content item by clicking the star icon next to the Content
 Bookmarks are stored per user. You can view a list of all bookmarks by clicking Bookmarks in the left menu.
 
 ![Bookmark screen](img/bookmarks.png)
+
+## Segments
+
+Segments enable you to target content at specific groups of your users.
+You can use them to display different content to different visitors to your Pages using the [Targeting block](working_with_page.md#targeting-block).
+
+You can create and configure Segments and Segment Groups they belong to in the **Admin** section of the Back Office.
```
Comment (DominikaK) [inline docs/site_organization/organizing_the_content.md:219]:
```suggestion
You can create and configure Segments and Segment Groups in the **Admin** section of the Back Office.
```

File: docs/site_organization/working_with_page.md (hunk 1 above)
Comment (DominikaK) [reply docs/site_organization/working_with_page.md:123]: No specific order necessary. The structure is the same as with other blocks, did not want to introduce a different organization.

**Review by dabrt — APPROVED** [review]

**Pull request merged**